gammapy/cube/psf_map.py

```diff
@@ -229,3 +313,32 @@ def containment_radius_map(self, energy, fraction=0.68):
             m.fill_by_coord(coord, containment_radius)
 
         return m
+
+    def stack(self, other):
+        """Stack PSFMap with another one.
+
+        This works only if the PSFMap to be stacked contain compatible exposure maps.
```

Comment:
More clearly state that this PSF is unchanged and a new PSF is returned?

Also the statement that exposure map geom has to be consistent: suggest to change it to just "geom has to be consistent". It also applies to the PSF value map geom, right?

Once there is a caller where many PSFs are stacked, we'll have to see if doing it one by one is convenient, or if a stack for an iterator of per-run PSFs would be nicer. Suggest to leave as-is for now.